# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/groff/hotfix-getopt.patch.musl
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- groff-1.23.0/src/libs/libgroff/getopt.c.vanilla	2025-06-05 22:42:21.461888909 +0200
+++ groff-1.23.0/src/libs/libgroff/getopt.c	2025-06-05 22:42:29.266000000 +0200
@@ -122,7 +122,7 @@
    whose names are inconsistent.  */
 
 #ifndef getenv
-extern char *getenv ();
+extern char *getenv (const char *);
 #endif
 
 #endif /* not __GNU_LIBRARY__ */
